pycross/private/tools/poetry_translator.py

```diff
@@ -178,14 +188,18 @@ def parse_file_info(file_info) -> PackageFile:
                     PoetryDependency(name=name, spec=spec, marker=marker)
                 )
 
+        files = [parse_file_info(f) for f in lock_pkg.get("files", [])]
```

Comment:
Files are with the packages now rather than being in the metadata at the bottom

Reply:
Can you add that context as a comment somewhere around here?